### PR TITLE
Fix/citadel tag

### PR DESCRIPTION
**Pull request by Guillaumebeuzeboc**

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: gazebo
-version: '0.1.0'
+version: '0.1.1'
```
Comment (giusebar) [inline snap/snapcraft.yaml:2]: why bumping to 0.1.1?

Comment (Guillaumebeuzeboc) [reply snap/snapcraft.yaml:2]: because I already tagged 0.1.0 and realized tags were not in the CI files

Comment (giusebar) [reply snap/snapcraft.yaml:2]: but I think that a change in the CI file should not affect the version of the snap?

Comment (Guillaumebeuzeboc) [reply snap/snapcraft.yaml:2]: This versioning being artificial, I don't think it matters so much. Plus, this way we don't remove the history by removing a tag and re-adding it.

**Review by giusebar — APPROVED** [review]

**Pull request merged**